### PR TITLE
Spelling: "owner", Log in, does not expire, your name

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: templates/goto.tmpl.php [hunk 1]
```diff
@@ -334,7 +334,7 @@
 			<?php if (User::isLoggedIn()) : ?>
 				<div class="notification">
 					<input type="checkbox" id="check_notif" class="checkbox" <?php if ($notification !== null) print_unescaped(' checked'); ?> />
-					<label for="check_notif"><?php p($l->t('Receive notification email on activity')); ?></label>
```
Comment (dartcafe) [inline templates/goto.tmpl.php]: We should keep this short for preventing line break in the mobile view.

Comment (splitt3r) [reply templates/goto.tmpl.php]: What about `Receive notifications per email`?

Comment (comradekingu) [reply templates/goto.tmpl.php]: Added another commit, and simplified this. :)

**Comment by splitt3r** [comment]
Looks good to me now 👍 what do you think @dartcafe?

**Pull request merged**